modules/apiserversource-kn.adoc

```diff
@@ -13,38 +13,51 @@ This guide describes the steps required to create, manage, and delete an ApiServ
 
 .Procedure
 
-. Create an ApiServerSource event source by entering the following command:
+. Create an ApiServerSource that uses a broker as an event sink, by entering the following command:
 +
+
+[source,terminal]
 ----
-$ kn source apiserver create testevents --sink broker:default --resource "event:v1" --service-account events-sa --mode Resource
+$ kn source apiserver create <event-source-name> --sink broker:<broker-name> --resource "event:v1" --service-account <service-account-name> --mode Resource
 ----
 
-. To check that the ApiServerSource is set up correctly, create a Knative service that dumps incoming messages to its log by entering the following command:
+. Create a Knative service that dumps incoming messages to its log, by entering the following command:
 +
+
+[source,terminal]
 ----
-$ kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
+$ kn service create <service-name> --image quay.io/openshift-knative/knative-eventing-sources-event-display:latest
 ----
 
-. Create a trigger from the `default` broker to filter events to the service created in the previous step by entering the following `kn` command:
+. Create a trigger to filter events from the `default` broker to the service, by entering the following command:
 +
+
+[source,terminal]
 ----
-$ kn trigger create event-display-trigger --sink svc:event-display
+$ kn trigger create <trigger-name> --sink svc:<service-name>
 ----
 
-. Create events by launching a pod in the default namespace. You can do this by entering the following command:
+. Create events by launching a pod in the default namespace, by entering the following command:
 +
+
+[source,terminal]
 ----
 $ oc create deployment hello-node --image=quay.io/openshift-knative/knative-eventing-sources-event-display
 ----
 
-. Check that the controller is mapped correctly by entering the following command and inspecting the output:
+. Check that the controller is mapped correctly, by entering the following command and inspecting the output:
```

Comment:
Deviating some from my previous comments to remove ", by entering the following command:" because it seems important that the user inspects the command output.

s/Check that the controller is mapped correctly, by entering the following command and inspecting the output:/Check that the controller is mapped correctly by inspecting the output to the following command:/